### PR TITLE
Feature/switch from uiwebview to wkwebview

### PR DESCRIPTION
**Pull request by tomerlevi444**
@lilach - please review :)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,21 +9,21 @@ install:
 - (cd integration-tests && bundle install)
 
 script:
+# Build & run tests
```
Comment (grozen) [inline .travis.yml]: This was after what is now line 16 on purpose. We didn't want the integration tests speaking with Ringcaptcha.

File: KlarnaOnDemand/KODWebViewController.m [hunk 2]
```diff
@@ -121,5 +115,30 @@ - (void)unregisterJockeyCallbacks {
   [Jockey off:JockeyUserError];
 }
 
+- (void)webView:(WKWebView *)webView didFinishNavigation:(WKNavigation *)navigation {
```
Comment (lilach) [inline KlarnaOnDemand/KODWebViewController.m]: You might want to move all the WKNavigationDelegate to one place, maybe even under a pragma mark

File: KlarnaOnDemand/KODRegistrationViewController.m [hunk 3]
```diff
@@ -43,9 +43,10 @@ - (NSURL *)url {
   return [KODUrl registrationUrlWithSettings: _registrationSettings];
 }
 
-- (void)webView:(UIWebView *)webView didFailLoadWithError:(NSError *)error {
-  [super webView:webView didFailLoadWithError:error];
-  
+
+-(void)webView:(WKWebView *)webView didFailNavigation:(WKNavigation *)navigation withError:(NSError *)error {
```
Comment (lilach) [inline KlarnaOnDemand/KODRegistrationViewController.m]: for the sake of consistency, add a space between `-` and `(void)`. same goes to line 40 in `KODPreferencesViewController.m`, and lines 22, 47, 53 in `KODWebViewController.m`.
also, there's an extra newline up here

File: KlarnaOnDemand/KODWebViewController.m (hunk 2 above)
Comment (tomerlevi444) [inline KlarnaOnDemand/KODWebViewController.m]: done

File: KlarnaOnDemand/KODRegistrationViewController.m (hunk 3 above)
Comment (tomerlevi444) [inline KlarnaOnDemand/KODRegistrationViewController.m]: done

File: .travis.yml (hunk 1 above)
Comment (tomerlevi444) [inline .travis.yml]: reverted.

**Comment by lilach** [comment]
lgtm

**Pull request merged**